packages/vue-apollo-composable/src/useApolloClient.ts

```diff
@@ -41,14 +41,14 @@ export function useApolloClient<TCacheShape = any>(clientId?: ClientId): UseApol
     }
   }
 
-  const providedApolloClients: ClientDict<TCacheShape> = inject(ApolloClients, null)
-  const providedApolloClient: ApolloClient<TCacheShape> = inject(DefaultApolloClient, null)
+  const providedApolloClients: ClientDict<TCacheShape> | null = inject(ApolloClients, null)
+  const providedApolloClient: ApolloClient<TCacheShape> | null = inject(DefaultApolloClient, null)
 
-  function resolveClient(id: ClientId = clientId) {
+  function resolveClient(id: ClientId | undefined = clientId) {
```

Comment:
Why the `undefined` here?

Reply:
Because `clientId` has type `string | undefined` and it cannot be assigned to type `ClientId` which is an alias for `string`